pkg/postgres/observationStore.go

```diff
@@ -22,7 +23,31 @@ func (o ObservationStore) GetObservation(id string) (*Observation, error) {
 	}
 	return &observation, nil
 }
+func (s ObservationStore) CheckPermissions(observationId string, userId string) (bool, error) {
 
+	var observation Observation
+
+	if err := s.Model(&observation).
+		Relation("Student").
+		Relation("Student.School.Users", func(q *orm.Query) (*orm.Query, error) {
+			return q.Where("user_id = ?", userId), nil
+		}).
+		Where("observation.id=?", observationId).
+		Select(); err == pg.ErrNoRows {
+		return false, nil
+	} else if err != nil {
+		return false, err
```

Comment:
We should wrap the error here. So that we'll get its stack trace when logged. Something like to this:
```suggestion
		return false, richErrors.Wrap(err, "failed checking user access to observation")
```
The general advice I've heard is to wrap errors that are being sent across packages such as this. So that we also get the information about the original package that gets the error (postgres), not only about the package that creates the error (go-pg).